release/models/system/openconfig-system.yang

```diff
@@ -908,6 +914,49 @@ module openconfig-system {
     }
   }
 
+  grouping system-macaddr-routing-config {
+    description
+      "Configuration data for system's routing MAC addresses.";
+
+    leaf routing-mac {
+      type oc-yang:mac-address;
+      description
+        "Any packets destined to this MAC address must be routed by the system,
```

Comment:
I'm a little confused at this statement -- should we say that these MACs MUST be accepted by the system and forwarded per normal operations? I think this is what the rest of the paragraph says, but it'd be good to be very explicit.